cmd/installer/main.go

```diff
@@ -44,6 +48,13 @@ func parseOptions(uninstall *bool, options *install.SafeOptions) *rest.Config {
 	}
 	log.SetLevel(loggerLevel)
 
+	switch pp := corev1types.PullPolicy(pullPolicy); pp {
+	case corev1types.PullAlways, corev1types.PullNever, corev1types.PullIfNotPresent:
+		options.OptionsCommon.PullPolicy = pp
+	default:
+		panic(fmt.Sprintf("invalid pull policy: %q", pullPolicy))
```

Comment:
Keep calm and don't panic 😄 
Return an error instead?

Reply:
We panic if the log level or etcd certs are incorrect so I did the same for the pull policy. I agree that panic is overkill and we should return an error but we can do that for everything in a followup.